sqlite3.go

```diff
@@ -1285,6 +1294,22 @@ func blob(stmt *C.sqlite3_stmt, i C.int, copy bool) []byte {
 	return nil
 }
 
+// SetErrorLogFunc registers a function that is invoked by SQLite whenever
+// an anomaly occurs. It is good practice to hook this up to your application's
+// logging framework, in order to aid debugging.
+// https://sqlite.org/errlog.html
+func SetErrorLogFunc(f ErrorLogFunc) error {
```

Comment:
Note that this signature is not consistent with the other similar functions, `Conn.CommitFunc`, `Conn.RollbackFunc` etc.:

- This function returns an `error`. The `sqlite3_config()` function can fail at least in principle, so it seems right to propagate that.
- This function does not return the previous callback (if any). I did not really see the value in that. Happy to add it though.
- I've chosen a name starting with `Set`, to give a better description of what this does.